app/screens/navigation.ts

```diff
@@ -875,3 +878,26 @@ export async function findChannels(title: string, theme: Theme) {
         options,
     );
 }
+
+export function goToPlaybookRuns(intl: IntlShape, channelId: string) {
+    const title = intl.formatMessage({id: 'playbooks.playbooks_runs.title', defaultMessage: 'Playbook runs'});
+    goToScreen(Screens.PLAYBOOKS_RUNS, title, {channelId}, {});
+}
+
+export async function goToPlaybookRun(intl: IntlShape, playbookRunId?: string, channelId?: string) {
+    const title = intl.formatMessage({id: 'playbooks.playbook_run.title', defaultMessage: 'Playbook run'});
+    let idToUse = playbookRunId;
+    if (!idToUse && channelId) {
+        const database = await DatabaseManager.getActiveServerDatabase();
+        if (database) {
+            const models = await queryActivePlaybookRunsPerChannel(database, channelId).fetch();
+            idToUse = models[0]?.id;
+        }
+    }
+
+    if (!idToUse) {
+        return;
+    }
+
+    goToScreen(Screens.PLAYBOOK_RUN, title, {playbookRunId: idToUse}, {});
+}
```

Comment:
not a fan of this, specially the use of the database being queried here, I would like to keep this as navigation only functions.

Reply:
I didn't want to get the "first playbook run" on the component, just when I really need it. That is why I put it here. I see two alternatives:
- Get the playbook run id in the index file of the component
- Create an action that gets the id and then calls this navigation function (which I feel is also a lot of boilerplate)

At the end I moved this to the component, but happy to discuss other options.